src/lizaalert/courses/models.py

```diff
@@ -71,6 +79,9 @@ class Course(TimeStampedModel):
     knowledge = models.ManyToManyField(Knowledge, through="CourseKnowledge", verbose_name="Умения", null=True)
     faq = models.ManyToManyField(FAQ, through="CourseFaq", verbose_name="Часто задаваемые вопросы", null=True)
     user_created = models.ForeignKey(User, on_delete=models.PROTECT, verbose_name="Создатель курса")
+    course_status = models.CharField(
```

Comment:
моделька уже называется Course, переименую в status

```suggestion
    status = models.CharField(
```